Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1359,6 +1359,14 @@ def generate_enabled_by_for_receiver(receiver, messages, ignore_invalid_message_
     ]
 
 
+def header_for_receiver_name(name):
+    special_headers = {
```

Comment:
You probably want to leave a comment here (why)? The mapping is very surprising and we should make it known that this is never the precedent to follow for the happy case.